src/templates/hi/2019/methodology.html

```diff
@@ -0,0 +1,428 @@
+          <source type="image/jpeg" srcset="/static/images/methodology-banner-sm.png" />
+          <img src="/static/images/methodology-banner.png" class="content-banner" alt="" width="1200" height="984" loading="eager" />
+        </picture>
+        <h2 id="overview"><a href="#overview" class="anchor-link">अवलोकन</a></h2>
+
+        <p>
+          Web Almanac <a href="https://httparchive.org">HTTP Archive</a> द्वारा आयोजित एक परियोजना है। HTTP Archive 2010 में स्टीव सॉडर्स द्वारा वेब को कैसे बनाया जाता है, इसे ट्रैक करने के लिए मिशन के साथ शुरू किया गया था। यह मासिक आधार पर लाखों वेब पेजों की संरचना का मूल्यांकन करता है और <a href="https://httparchive.org/faq#how-do-i-use-bigquery-to-write-custom-queries-over-the-data">BigQuery</a> पर विश्लेषण के लिए टेराबाइट्स मेटाडेटा उपलब्ध कराता है। <a href="https://httparchive.org/about">HTTP Archive के बारे में</a> अधिक जानें।
+        </p>
+
+        <p>
+          Web Almanac का मिशन HTTP Archive के डेटा वेयरहाउस को वेब समुदाय के लिए और अधिक सुलभ बनाना है, जिसमें हम विषय वस्तु विशेषज्ञ की प्रासंगिक जानकारी प्रदान करते हैं। आप इसे वेब की स्थिति के बारे में ज्ञान के वार्षिक भंडार के रूप में सोच सकते हैं, 2019 इसका पहला संस्करण होगा।
+        </p>
+
+        <p>
+          Web Almanac के 2019 संस्करण में चार स्तंभ शामिल हैं: सामग्री, अनुभव, प्रकाशन और वितरण। लिखित रिपोर्ट का प्रत्येक भाग एक स्तंभ का प्रतिनिधित्व करता है और इसके विभिन्न पहलुओं की खोज करने वाले अध्यायों से बना है। उदाहरण के लिए, भाग II यूजर एक्सपीरियंस का प्रतिनिधित्व करता है और इसमें प्रदर्शन, सुरक्षा, पहुंच, SEO, PWA और मोबाइल वेब अध्याय शामिल हैं।
+        </p>
+    </section>
+
+    <section>
+        <h2 id="dataset"><a href="#dataset" class="anchor-link">डेटासेट के बारे में</a></h2>
+
+        <p>
+          HTTP Archive डेटासेट लगातार नए मासिक डेटा के साथ अपडेट हो रहे है। Web Almanac के 2019 संस्करण के लिए, जब तक कि अन्यथा अध्याय में उल्लेख नहीं किया गया, जुलाई 2019 से सभी मैट्रिक्स क्रॉल किए गए थे। ये परिणाम <code>2019_07_01</code> के साथ उपसर्ग किए गए तालिकाओं में BigQuery पर <a href="https://github.com/HTTPArchive/httparchive.org/blob/master/docs/gettingstarted_bigquery.md">सार्वजनिक रूप से क्वेरी करने योग्य</a> हैं।
+        </p>
+
+        <p>
+          Web Almanac में प्रस्तुत सभी मैट्रिक्स BigQuery पर डेटासेट का उपयोग कर सार्वजनिक रूप से प्रतिलिपि प्रस्तुत करने योग्य हैं। आप हमारी <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2019">GitHub रिपॉजिटरी</a> में सभी अध्यायों द्वारा उपयोग किए गए क्वेरी को देख सकते हैं।
+        </p>
+
+        <p class="note">
+          कृपया ध्यान दें कि इनमें से कुछ क्वेरी काफी बड़ी हैं और आप के प्रयोग के लिए <a href="https://cloud.google.com/bigquery/pricing">महंगे</a> हो सकते हैं, क्योंकि BigQuery को टेराबाइट द्वारा बिल किया जाता है। अपने खर्च को नियंत्रित करने में सहायता के लिए, टिम कडलेक की पोस्ट <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a> देखें।
```

Comment:
I get a weird message when I use the [HTML validator](https://validator.w3.org/nu/) on this line:

![Nu validation message](https://user-images.githubusercontent.com/10931297/99961890-1de03580-2d87-11eb-8b67-a363e96edb81.png)

Which apparently [is some encoding issue](https://stackoverflow.com/a/5466610/2144578).

I ran that text through a normaliser and got this and no it doesn't complain. Is this the same?

```suggestion
          कृपया ध्यान दें कि इनमें से कुछ क्वेरी काफी बड़ी हैं और आप के प्रयोग के लिए <a href="https://cloud.google.com/bigquery/pricing">महंगे</a> हो सकते हैं, क्योंकि BigQuery को टेराबाइट द्वारा बिल किया जाता है। अपने खर्च को नियंत्रित करने में सहायता के लिए, टिम कडलेक की पोस्ट <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a> देखें।
```

Reply:
Fixed